src/types.ts

```diff
@@ -98,4 +109,8 @@ export interface Redis {
   scriptLoad: (
     ...args: [script: string]
   ) => Promise<string>;
+
+  smismember: (
+    key: string, members: string[]
+  ) => Promise<(0 | 1)[]>;
```

Comment:
Maybe we can put another branded type here.

Reply:
added IsMember